dev-tools/reconfigurator-cli/src/lib.rs

```diff
@@ -372,6 +377,31 @@ enum BlueprintEditCommands {
     },
     /// expunge a zone
     ExpungeZone { zone_id: OmicronZoneUuid },
+    /// configure an SP update
+    SpUpdateSet {
```

Comment:
This is the only command here where the action (verb) comes after the subject (noun). I think we should be consistent and change this to `SetSpUpdate`. 

This has a side benefit. When I first read this, I thought you were declaring an object called in "sp update set", but I couldn't figure out what the set contained.

Reply:
Yeah, fair enough.  I much prefer `subject-verb[-object]`, but it isn't what's here, and consistency is better here.  Good catch!